sudachipy/plugin/oov/mecab_oov_plugin.py

```diff
@@ -32,9 +32,10 @@ def __init__(self):
 
     class OOV:
         def __init__(self):
-            self.left_id = None
-            self.right_id = None
-            self.cost = None
+            #XXX Is this safe?
```

Comment:
This is fine, could you delete this comment line?

Reply:
Ah, sorry, forgot to remove that after I checked it.